modules/ROOT/pages/manage-dependency-versions-design-center.adoc

```diff
@@ -1,12 +1,14 @@
-= Update Dependencies in a Mule-Application Project (Design Center)
+= Update Dependencies in a Mule-Application Project (Flow Designer)
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 :imagesdir: ../assets/images
 :keywords: mozart
 
-Dependencies are connectors and other parts of a flow that are versioned independently from the Mule runtime. You can update an existing project to use a different version of a dependency without losing its configuration. However, if the change to the version affects configuration fields, you might need to change those settings as described in the release notes.
+Dependencies are connectors and other parts of a flow that are versioned independently from Mule Runtime. For example, suppose you are using the Sharepoint connector in a flow; although you might be using Mule Runtime 4.1.5, you might be using version 3.1.1 of the Sharepoint connector. You can switch to a different version of a dependency. For example, you might want to switch to the latest version of a connector, so that your Mule application can use a new feature that is included in that version.
```

Comment:
According to our glossary, it's either Mule or Mule runtime engine, but never Mule Runtime. (I realize it's controversial.)

Second sentence, I suggest "For example, if you are using Sharepoint connector 3.1.1 in a flow and Mule 4.1.5, you can switch to a different version of a dependency. (feel free to disregard if you disagree)

In the last sentence, the second comma isn't necessary.

Reply:
Thanks for the help with that sentence. The paragraph now reads: 
_Dependencies are connectors and other parts of a flow that are versioned independently from Mule. For example, if you are using Sharepoint connector 3.1.1 in a flow and Mule 4.1.5, you can switch to a different version of the connector and continue to use Mule 4.1.5. You might want to switch to the latest version of a connector so that your Mule application can use a new feature that is included in that version._